app/react-components/components/DownloadVisualizationButton.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import DropDownButton from './DropDownButton';
+import downloadSvg from '../utils/download-svg';
+import saveFile from '../utils/filesaver';
+import toTsvString, {mapArrayToTsvString} from '../utils/toTsvString';
+
+function getSelector(el) {
+  return typeof el === 'string' ? document.querySelector(el) : el;
+}
+const styles = {
+  hidden: {
+    width: 0,
+    height: 0,
+    overflow: 'hidden',
+  },
+};
+
+const icon = <i className={"fa fa-download"}/>;
```

Comment:
Could we actually try to avoid react-icons altogether? it still doesn't build with `NODE_ENV=production` fluent in expressjs